map_server:
  ros__parameters:
    yaml_filename: '/path/to/your/map/yaml/path'

waypoint_creater_node:
  ros__parameters:
    waypoints_file_name : 'test.csv'

